daal4py/sklearn/manifold/__init__.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python
 #===============================================================================
-# Copyright 2014 Intel Corporation
+# Copyright 2020 Intel Corporation
```

Comment:
2020-2022 ?

Reply:
Now we have to specify only the year of adding the file

Reply:
we can use oldest date only - there is no need for interval. 

So date should reflect year of file/content creation